core/src/main/groovy/noe/workspace/WorkspaceHttpdBaseOS.groovy

```diff
@@ -0,0 +1,51 @@
+package noe.workspace
+
+import groovy.util.logging.Slf4j
+import noe.common.utils.Library
+import noe.rhel.server.httpd.HttpdBaseOS
+
+@Slf4j
+class WorkspaceHttpdBaseOS extends WorkspaceAbstract {
+
+  WorkspaceHttpdBaseOS() {
+
+    if (platform.isRHEL8()) {
```

Comment:
What is the difference between R7 and R8? This seems like premature 'optimization'. Ot why we need branch for R7 and R8?

Reply:
you are right, premature optimization. I thought it would be nice, but no real use for now. Removing